src/main/java/org/fenixedu/academic/domain/Degree.java

```diff
@@ -316,6 +316,30 @@ public DegreeCurricularPlan createPreBolonhaDegreeCurricularPlan(String name, De
         }
     }
 
+    @Deprecated
+    public DegreeCurricularPlan createDegreeCurricularPlan(String name, GradeScale gradeScale, Person creator) {
```

Comment:
What is the point on creating a new already-deprecated method in a core domain class, when its sole purpose is to be used in a service?